shared/profile/friendships.native.js

```diff
@@ -1,19 +1,89 @@
+import {Box, Avatar, Text} from '../common-adapters'
+import TabBar, {TabBarItem} from '../common-adapters/tab-bar'
+import {globalStyles, globalColors} from '../styles/style-guide'
+import type {Props, UserInfo} from './friendships'
 
-class Render extends Component<void, Props, void> {
-  _renderComingSoon () {
-    return <ComingSoon />
-  }
+type UserEntryProps = UserInfo & {
+  onClick?: (username: string) => void
+}
+
+const UserEntry = ({onClick, username, followsYou, following}: UserEntryProps) => (
+  <Box style={userEntryContainerStyle} onClick={() => { onClick && onClick(username) }}>
+    <Avatar style={userEntryAvatarStyle} size={48} username={username} followsYou={followsYou} following={following} />
+    <Text type='BodySmall' style={userEntryUsernameStyle(followsYou)}>{username}</Text>
+  </Box>
+)
+
+const userEntryContainerStyle = {
+  ...globalStyles.clickable,
+  ...globalStyles.flexBoxColumn,
+  alignItems: 'center',
+  justifyContent: 'flex-start',
+  width: 105,
+  padding: 2
+}
+
+const userEntryAvatarStyle = {
+  marginBottom: 2,
+  marginTop: 5
+}
 
+const userEntryUsernameStyle = followsYou => ({
+  color: followsYou ? globalColors.green : globalColors.blue,
+  textAlign: 'center'
+})
+
+class Render extends Component<void, Props, void> {
   render () {
-    if (this.props.showComingSoon) {
-      return this._renderComingSoon()
-    }
-    return <Box />
+    return (
+      <TabBar tabWidth={110}>
+        <TabBarItem
+          selected={this.props.currentTab === 'FOLLOWERS'}
+          label={'FOLLOWERS'}
+          onClick={() => { this.props.onSwitchTab && this.props.onSwitchTab('FOLLOWERS') }}>
+          <Box style={tabItemContainerStyle}>
+            <Box style={tabItemContainerTopBorder} />
+            <Box style={tabItemContainerUsers}>
+              {this.props.followers.map(user => <UserEntry key={user.username} {...user} onClick={this.props.onUserClick} />)}
+            </Box>
+          </Box>
+        </TabBarItem>
+        <TabBarItem
+          selected={this.props.currentTab === 'FOLLOWING'}
+          label={'FOLLOWING'}
+          onClick={() => { this.props.onSwitchTab && this.props.onSwitchTab('FOLLOWING') }}>
+          <Box style={tabItemContainerStyle}>
+            <Box style={tabItemContainerTopBorder} />
+            <Box style={tabItemContainerUsers}>
+              {this.props.following.map(user => <UserEntry key={user.username} {...user} onClick={this.props.onUserClick} />)}
+            </Box>
+          </Box>
+        </TabBarItem>
+      </TabBar>
+    )
   }
 }
 
+const tabItemContainerStyle = {
+  ...globalStyles.flexBoxColumn,
+  flex: 1,
+  backgroundColor: globalColors.lightGrey
+}
+
+const tabItemContainerTopBorder = {
+  flex: 1,
+  height: 1,
+  backgroundColor: globalColors.black_10,
+  alignSelf: 'stretch'
+}
+
+const tabItemContainerUsers = {
+  ...globalStyles.flexBoxRow,
+  flex: 1,
+  flexWrap: 'wrap',
+  justifyContent: 'space-around'
```

Comment:
Spacing around so that the items always appeared center independent of how wide the device is. This looks weird when the rows change from having 3 entries to only having 1 or 2, but I think it's a price worth paying so that all the other rows look better